src/components/ConfigProvider/ConfigProvider.tsx

```diff
@@ -11,6 +18,8 @@ const DEFAULT_FONT_SIZE = 16;
 
 const ConfigProvider: FC<ConfigProviderProps> = ({
     children,
+    focusVisible = false,
+    focusVisibleElement = document.documentElement,
```

Comment:
can focus visiblie have its own interface like themeoptions, will make this a bit cleaner